authz/providers/azure/rbac/rbac.go

```diff
@@ -373,6 +377,11 @@ func (a *AccessInfo) CheckAccess(request *authzv1.SubjectAccessReviewSpec) (*aut
 }
 
 func (a *AccessInfo) sendCheckAccessRequest(ctx context.Context, checkAccessUsername string, checkAccessURL url.URL, checkAccessBody *CheckAccessRequest, ch chan *authzv1.SubjectAccessReviewStatus) error {
+	var clientInCtx *http.Client
+	if c, ok := ctx.Value(oauth2.HTTPClient).(*http.Client); ok {
```

Comment:
nit: can we move this client loader to `util/azure` so we can hide the context key from caller?